packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,91 @@
+* The End User provides proof of identity (using any of the supported authentication methods) and gives consent for any requested claims defined in your `scopes` parameter
+* The browser receives an authorization code response from the Okta Authorization Server. If there is an error or the authentication fails for any reason, the error returned to the End User through the browser. If the authentication is successful, the Authorization Server issues an authorization `code` value and includes it in a redirect that sends the browser over to the Client at the Login Redirect URI callback location.
+* The Client (your application) then submits a request to the Okta /token endpoint using the authorization code and the Client Secret. If the code and secret are both valid, then Okta returns:
+  * `access_token` &mdash; this token is restricted to give access only to what the End User authorized for your application
+  * `token_type=Bearer` &mdash; this identifies the authentication method used
+  * `expires_in` &mdash; the expiration time for the access token (in seconds)
+  * `scope` &mdash; the scopes that are contained in the access token
+  * `id_token` &mdash; the ID token contains the specific info (in a JWT format) about your user that is needed by the application
+  * Optional. `refresh_token` &mdash; this token is returned if the `offline_access` scope is granted
+* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the End User.
+
+The Authorization code process flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
+
+Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores or an End User could view and possibly modify your code. For this reason, OIDC native, mobile, and single page application (SPA) app integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
+
+To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a custom authorization server, including the `default` server.
+
+A full guide to setting up the application to use the Authorization Code Flow can be found in our Okta developer guide: Implement the Authorization Code Flow
+
+Also, if you’ve used SAML for SSO in the past, it’s important to realize that the OAuth/OIDC flow is different, because OAuth/OIDC is not just an assertion that is exchanged between Okta and your SaaS back-end, but a long-term token that can be used for callback into Okta at any point as long as the token is valid.
```

Comment:
```suggestion
Also, if you’ve used SAML for SSO in the past, it’s important to realize that the OIDC flow is different, because the OIDC protocol does not just provide an assertion that is exchanged between Okta and your SaaS back-end, but a long-term token that can be used for callback into Okta at any point as long as the token is valid.
```

Reply:
Changed to:
> Also, if you’ve used SAML for SSO in the past, it’s important to realize that the OIDC flow is different. The OIDC protocol doesn't just provide an assertion that is exchanged between Okta and your SaaS back-end, but uses a long-term token that can be used for callback into Okta at any point as long as the token is valid.